index.js

```diff
@@ -214,6 +239,54 @@ const lookUpCommand = (message) => {
     });
 };
 
+/*
+
+  const topFiveCommand = (message) => {};
+    The user uses this command to select up to 5 cards  from their inventory to display
+    any user can use the lookup command to look at another users top 5 cards, but only the owner of cards can select which cards go into their top 5
+    
+    To input a card to your top 5, the command must be entered as !showcase blue-eyes white dragon 
+
+
+
+*/
+
+const removeCardCommand = (message) => {
+  const removeThisCard = message.content.split(" ").slice(1).join(" ");
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    const userIdentification = message.author.id;
+    const userInventory = userData[userIdentification].inventory;
+    const inventoryRef = ref.child(message.author.id).child("inventory");
+
+    const cardIDToRemove = Object.keys(userInventory).find(
+      (key) =>
+        userInventory[key].name.toLowerCase() === removeThisCard.toLowerCase()
+    );
+    console.log(cardIDToRemove);
```

Comment:
Consider either removing some of the many `console.log`s, or providing some more context in the message, so that when you see them on Render, you know what they are.

What I usually do is something like: `console.log("Removing card", cardIDToRemove);` -- the comma puts stuff on a new line for `console.log`s